### PR TITLE
#601 - Add note for usage of multiple value search in Filters

### PR DESCRIPTION
**Pull request by Tim59**
#601 - Add note for usage of multiple value search in Filters

### CODE REVIEW

File: core/filters.md [hunk 1]
```diff
@@ -136,6 +136,10 @@ Case insensitivity may already be enforced at the database level depending on th
 used. If you are using MySQL, note that the commonly used `utf8_unicode_ci` collation (and its sibling `utf8mb4_unicode_ci`)
 are already case insensitive, as indicated by the `_ci` part in their names.
 
+Note : Search filters with the exact strategy can have multiple values for a same property (in this case the condition will be similar to a SQL IN clause)
```
Comment (alanpoulain) [inline core/filters.md]: Note : => Note:

Comment (alanpoulain) [reply core/filters.md]: You can add a final `.`.

**Comment by alanpoulain** [comment]
Thank you @Tim59!

**Comment by Tim59** [comment]
You'e welcome :)

**Pull request merged**